profile/README.md

```diff
@@ -0,0 +1,27 @@
+# Welcome to Nebari! 👋
+
+<a href="https://nebari.dev">
+    <img src="nebari-banner.svg" alt="A banner showing the Nebari logo witt the tagline 'A data science platform: Built for scale, designed for collaboration.' This is followed by a get started button pointing to Nebari 101"/>
```

Comment:
For reference: The banner is created using [Google Slides here](https://docs.google.com/presentation/d/1_X-EcpFNzrMSyO74oBBi0do_4bsTQwS5xciLOTLbikY/edit?usp=sharing). Please feel free to comment on the slide directly if needed!

Reply:
I think we can make the banner shorter in height.